backend/app/images.js

```diff
@@ -15,9 +15,16 @@ const Response = require('/opt/nodejs/response.js')
 
 exports.get = async (event, context) => {
   let response = new Response(event)
+
+  let size = event.queryStringParameters['size'] : '1'
+  // This If statement is here solely to prevent any sort of injection. We always convert user input into something safe to run in aws.
+  // Additionally, sizes of 1 and 2 were previously used for this api endpoint. The if statement is necessary for maintaining the legacy interface.
```

Comment:
I don't think there is a risk for injection here. AWS is preprocessing the request so it should be okay, the incoming data will be a string. I think the risk comes if we do something extremely weird like running JSON.parse() over the query input.

As it is now, it is slightly confusing the way size is being manipulated. It should probably just be one if statement creating the "pre-path."

`
let prepath = '/fullpath/'
if (event.queryStringParameters['size'] && parseInt(event.queryStringParameters['size']) === 2) {
  prepath = '/thumbnails/'
}
`